filebeat/module/haproxy/log/ingest/pipeline.json

```diff
@@ -29,6 +30,26 @@
                 ]
             }
         },
+        {
+            "date": {
+                "formats": [
+                    "dd/MMM/yyyy:HH:mm:ss.SSS",
+                    "MMM dd HH:mm:ss"
+                ],
+                "timezone" : "{{ event.timezone }}",
+                "on_failure" : [
+                    {
+                        "append" : {
+                            "field" : "error.message",
+                            "value" : "{{ _ingest.on_failure_message }}"
+                        }
+                    }
+                ],
+                "if" : "ctx.event.timezone != null",
```

Comment:
This should be at line 35? Like https://github.com/elastic/beats/blob/master/filebeat/module/system/syslog/ingest/pipeline.json#L46